### PR TITLE
Feature/contact card

### LINKED ISSUE #10: Create Contact card component
### Acceptance criteria's:
User is able to see contact card of the added contact on the URL "/".

The "Contact card" component should have next info:

- Contact Avatar;
- Name&Surname;
- Mobile phone;
- Label "Favorite" (if contact is marked as favorite);
- Job title;
- Position;

User is able to change the resolution of the screen and get mobile/desktop version. See *Figma (design) for media breakpoint and *Figma for details about design.

### Estimation: 6h

**Links:**
Figma design - [Figma](https://www.figma.com/file/J5Syu1z6LPJhIQdScukaa3/CoKeeper?node-id=0%3A1)
SRS - [SRS](https://docs.google.com/document/d/1Z0ceG_NtqwGXs6xRjqq8hb9AoqWj1lLzboL5epCSfkQ/edit#)

### PR DESCRIPTION
**Pull request by solomiia-sahala**
added contact card component with desktop/mobile styling for card view

### CODE REVIEW

File: src/components/ContactCard.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import '../styles/ContactCard.scss';
+
+const ContactCard = (props: any) => {
+    const favoriteIcon = "./images/favorite.svg";
+    const { nameSurname, position, job, mobile, favorite } = props.userContact;
```
Comment (chiffa-fox) [inline src/components/ContactCard.tsx]: If userContact is the only field you need from props you could destructure it right on enter
`const ContactCard = ({ userContact }: Props) => {`
But to do this you'll need to define a Props interface first:
```
interface Props {
  userContact: { 
  nameSurname: string
...
  }
}
```

File: src/components/Contacts.tsx [hunk 2]
```diff
@@ -1,7 +1,40 @@
...
+
+
+]
+
 const Contacts = () => {
     return (
         <div>
-            Contacts
+            {mockUserData.map(user => <ContactCard key={user.id} userContact={user} />
+
```
Comment (chiffa-fox) [inline src/components/Contacts.tsx]: This line looks extra here

**Review by chiffa-fox — CHANGES_REQUESTED** [review]

File: src/components/Contacts.tsx (hunk 2 above)
Comment (solomiia-sahala) [reply src/components/Contacts.tsx]: removed extra line.

File: src/components/ContactCard.tsx (hunk 1 above)
Comment (solomiia-sahala) [reply src/components/ContactCard.tsx]: yes, only userContact data used in 'render' method. Already added in a new request, thanks.

**Review by chiffa-fox — APPROVED** [review]

**Pull request merged**